drivers/adcxx1c/adcxx1c.c

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (C) 2017 OTA keys S.A.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_adcxx1x
+ *
+ * @file
+ * @brief       ADCXX1C ADC device driver
+ *
+ * @author      Vincent Dupont <vincent@otakeys.com>
+ */
+
+#include "adcxx1c.h"
+#include "adcxx1c_params.h"
+
+#include "periph/i2c.h"
+#include "periph/gpio.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+#define I2C_SPEED I2C_SPEED_FAST
+
+#define I2C (dev->params.i2c)
+#define ADDR (dev->params.addr)
+
+#define ADCXX1C_CONV_RES_ADDR      (0)
```

Comment:
these should go into the header file, or not?  Could also be an anonymous `enum`.